tests/test_project.py

```diff
@@ -539,6 +643,29 @@ def test_load_invalid_schema(project):
     assert "invalid" in str(excinfo.value)
 
 
+def test_load_module_project_succeeds(project):
+    project.artifact_type = "MODULE"
+    project.type_name = "Unit::Test::Malik::MODULE"
+    patch_load_settings = patch.object(
+        project, "load_settings", return_value={"artifact_type": "MODULE"}
+    )
+    with patch_load_settings:
+        project.load()
```

Comment:
This test leaks a `schema.json` file in the root directory

Reply:
This has been fixed since, btw